addons/godot_rl_agents/sensors/sensors_3d/RGBCameraSensor3D.gd

```diff
@@ -8,4 +8,4 @@ func get_camera_pixel_encoding():
 	return camera_texture.get_texture().get_image().data["data"].hex_encode()
 
 func get_camera_shape()-> Array:
-	return [$SubViewport.size[0], $SubViewport.size[1], 4]
+	return [$SubViewport.size[0], $SubViewport.size[1], 3]
```

Comment:
SubViewport does have a property `transparent_bg` which adds an alpha channel. Default is false. 

![image](https://github.com/edbeeching/godot_rl_agents_plugin/assets/1820121/50f85218-c416-40ee-856e-a4ae04d0042d)


We can check that value to avoid errors:

```suggestion
	if $SubViewport.transparent_bg:
		return [$SubViewport.size[0], $SubViewport.size[1], 4]
	else:
		return [$SubViewport.size[0], $SubViewport.size[1], 3]
```

Reply:
Great addition, thanks.